public/js/collections/CourseSearch.js

```diff
@@ -0,0 +1,12 @@
+// coursesearch.js
+// --------
+define(['jquery', 'underscore', 'backbone'], function($, _, Backbone) {
```

Comment:
We removed requirejs since none of us had really used it and it was just something else to learn that didn't add functionality.